collector/systemd_linux.go

```diff
@@ -123,15 +131,15 @@ func NewSystemdCollector(logger log.Logger) (Collector, error) {
 		"Detected systemd version", []string{}, nil)
 
 	if *oldUnitExclude != "" {
-		if *unitExclude == "" {
+		if !unitExcludeSet {
```

Comment:
Why not `if unitExclude != nil && *unitExclude == ""` instead of having the additional var?

Reply:
Because there is a default value

Reply:
Ah I see, ugh.

Reply:
We use this in promu.